### DISCUSSION TITLE
Error: Image must have at least 1 frame to write a bitmap in plotMarkerHeatmap

### DISCUSSION THREAD

**Discussion by khsjh** [GreenleafLab/ArchR#1948] [Questions / Documentation]
When I running chapter 11.2 Plotting Marker Peaks in ArchR in full manual, I got a error message

`Error: Image must have at least 1 frame to write a bitmap in plotMarkerHeatmap`

My running code is almost same to manual

```
markersPeaks <- getMarkerFeatures(
  ArchRProj = projHeme5, 
  useMatrix = "PeakMatrix", 
  groupBy = "Clusters2",
  bias = c("TSSEnrichment", "log10(nFrags)"),
  testMethod = "wilcoxon"
)

> markersPeaks
class: SummarizedExperiment 
dim: 156653 11 
metadata(2): MatchInfo Params
assays(7): Log2FC Mean ... AUC MeanBGD
rownames(156653): 1 2 ... 156652 156653
rowData names(4): seqnames idx start end
colnames(11): B CD4.M ... PreB Progenitor
colData names(0):

heatmapPeaks <- plotMarkerHeatmap(
  seMarker = markersPeaks, 
  cutOff = "FDR <= 0.1 & Log2FC >= 1.5",
  transpose = TRUE
)
draw(heatmapPeaks, heatmap_legend_side = "bot", annotation_legend_side = "bot")
```

I also attempt my log file here
[ArchR-plotMarkerHeatmap-fb9e854bfad6-Date-2023-05-16_Time-15-34-44.log](https://github.com/GreenleafLab/ArchR/files/11484791/ArchR-plotMarkerHeatmap-fb9e854bfad6-Date-2023-05-16_Time-15-34-44.log)

**Comment by Jyoung-0108** [comment]
hello, i encountered the same error when running plotMarkerHeatmap, have you figured it out? Thanks a lot!

**Comment by NayeliGutierrez** [comment]
Same error with tutorial dataset. Did you find any solution?

**Comment by khsjh** [comment]
Well..., as I guess, it will be solved if you reset the Rstudio memory.
Clean your all object what used in ArchR.
This problem was so long ago that I don't remember exactly how to solve it.

2024년 11월 1일 (금) 오전 2:48, Nayeli Gutierrez ***@***.***>님이 작성:

**Comment by joeyQX** [comment]
the same error!
